dev-libs/hyperscan/Manifest

```diff
@@ -1 +1,2 @@
+DIST hyperscan-4.4.1.tar.gz 1515106 SHA256 3a082d92a3cb0cd724bc1190d24cc39752bd3db35d22115fda03d2e91ccd94cd SHA512 5e3463061110f81ad145dccf566f8308eed7bfd6711af59461b59903dd77b20963df3dd0f0ec8fc925837e76b658352b420b1ce210c4c91a4cb128a4d5f83056 WHIRLPOOL 1eff40644b4284615f8011e4fd516005381468d651a2675c0e4b46b2b34a9986a5c242590314ef04ebe8528e6ee9090c07830d868de6fed84a0c244ad5ebf28d
 DIST v4.4.1.tar.gz 1515106 SHA256 3a082d92a3cb0cd724bc1190d24cc39752bd3db35d22115fda03d2e91ccd94cd SHA512 5e3463061110f81ad145dccf566f8308eed7bfd6711af59461b59903dd77b20963df3dd0f0ec8fc925837e76b658352b420b1ce210c4c91a4cb128a4d5f83056 WHIRLPOOL 1eff40644b4284615f8011e4fd516005381468d651a2675c0e4b46b2b34a9986a5c242590314ef04ebe8528e6ee9090c07830d868de6fed84a0c244ad5ebf28d
```

Comment:
Why is the distfile without package name still in this Manifest? Is there a hyperscan-4.4.1.ebuild which also needs adjustment?